packages/tailscale-pikvm/tailscale-pikvm.install

```diff
@@ -1,8 +1,15 @@
+pre_upgrade() {
+	service_enabled=$(systemctl is-enabled tailscaled)
+}
+
 post_install() {
 	post_upgrade
 }
 
 post_upgrade() {
 	mkdir -p /var/lib/tailscale || true
 	chmod 700 /var/lib/tailscale || true
+	if [ "$service_enabled" == "enabled" ]; then
+		systemctl enable --now tailscaled
```

Comment:
remove --now, change this line like `systemctl enable tailscaled || true`

Reply:
Updated